integration-cli/fixtures.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
This package is already pretty massive.

What do you think about putting these into `pkg/integration/fixtures/` instead?